# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/CredentialsContainer.h
# Commit: 63d9972bab1f
# Full Hash: 63d9972bab1f267a940ec30c7cea7a4c41e54fa6
# Author: Alexandru Marc <amarc@mozila.com>
# Date: 2025-06-06 08:52:03
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing leaks @ mochitest.toml
#   
#   This reverts commit a97cb1d6c08bc38e8644bfc916298b847dbac3a1.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r 801e25cb4bfd -r 63d9972bab1f dom/credentialmanagement/CredentialsContainer.h
--- a/dom/credentialmanagement/CredentialsContainer.h	Thu Jun 05 18:51:36 2025 +0000
+++ b/dom/credentialmanagement/CredentialsContainer.h	Thu Jun 05 21:40:49 2025 +0300
@@ -47,6 +47,7 @@
 
   nsCOMPtr<nsPIDOMWindowInner> mParent;
   RefPtr<WebAuthnHandler> mWebAuthnHandler;
+  bool mActiveIdentityRequest;
 };
 
 }  // namespace mozilla::dom